DataProc/PPass/main_rec.C

```diff
@@ -8,6 +8,14 @@ void main_rec(const char *filename="raw.root", const char* options="")
 
   
   AliReconstruction rec;
+
```

Comment:
I have noticed that current reconstruction macros for PPass do not allow to reconstruct only part of the chunk (unlike CPass0 and 1). This variable (not used on the Grid) adds this capability consistently for every reconstruction job.